main.js

```diff
@@ -0,0 +1,18 @@
+var express = require("express"),
+	app     = express()
+
+app.use(express.json())
+app.use(express.urlencoded({ extended: true }))
+
+var port = process.env.HDD_SUPERVISOR_PORT || 9090,
+	ip   = process.env.IP   || process.env.OPENSHIFT_NODEJS_IP || "0.0.0.0"
+
+var routes = require("./node_modules/data-api/src/api/routes")(app)
```

Comment:
./node_modules/data-api/src/api/routes suffices or js extension(to complete full file path) is required?

Reply:
Not required. Works for me.